### PR TITLE
improvements: added guest user name for displaying off-platform users name

### PR DESCRIPTION
**Pull request by BALAJI-BALA**

### CODE REVIEW

**Comment by mikevic** [comment]
@BALAJI-BALA - it's going to be hard for the UI to determine when to show either a user's name or a guest's name. Why don't we give them a field that will return that calculated?

**Comment by BALAJI-BALA** [comment]
> @BALAJI-BALA - it's going to be hard for the UI to determine when to show either a user's name or a guest's name. Why don't we give them a field that will return that calculated?

@mikevic Added a new field will return the calculated name.

File: app/graphql/types/objects/consultation_response/base.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ def visible?(context)
 						super && context[:current_user].present?
 					end			
 				end
+				field :user_name,													String, nil, null: true
```
Comment (mikevic) [inline app/graphql/types/objects/consultation_response/base.rb:28]:
```suggestion
				field :respondent_name,													String, nil, null: true
```

**Review by mikevic — CHANGES_REQUESTED** [review]
@BALAJI-BALA - minor change, you can merge this PR.

**Pull request merged**